test/components/grouping_stage_idea_card_test.js

```diff
@@ -11,7 +12,10 @@ describe("<GroupingStageIdeaCard />", () => {
   describe("when the given idea has coordinates", () => {
     before(() => {
       idea = { id: 5, body: "hello", x: 0, y: 109 }
-      wrapper = shallow(<GroupingStageIdeaCard idea={idea} />)
+      wrapper = shallow(
+        <GroupingStageIdeaCard idea={idea} actions={{}} />,
+        { disableLifecycleMethods: true }
```

Comment:
necessary due to the removal of a no-longer-necessary guard clause in `componentDidMount`